### DISCUSSION TITLE
help!!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#669] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i dont have access to the issues thing because my school blocks emails from things that arent from the district so could you dm it to me or send it via outlook

**Comment by ILikeBypassing** [comment, marked as answer]
p e r s o n a l d e v i c e

**Reply by Hilololsusy** [reply]
NO WAY! PERSONAL DEVICE!>!?!??!?!? BRO! I NEVER WOULD HAVE FIGURED THAT ONE OUT! bro. if i had a personal device why woudl I be on this shi

**Reply by AshtonDavies** [reply]
Because you wouldn't be bringing your mom's PC tower or whatever it is to school, that's why. Not all personal devices leave the home. Just get something to view it on other than your Chromebook is what they mean.
